src/devices/huntsman_v2_tkl.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Razer Huntsman V2 TKL",
+  "productId": "0x026b",
+  "mainType": "keyboard",
+  "image": "https://assets3.razerzone.com/02qciF5cWDrRcrL9CxzbHC017No=/500x500/https%3A%2F%2Fhybrismediaprod.blob.core.windows.net%2Fsys-master-phoenix-images-container%2Fhb8%2Fhd7%2F9239675600926%2Fhuntsman-v2-tkl-500x500.png",
```

```suggestion
  "image": "https://dl.razerzone.com/src/5638/5638-1-en-v1.png",
```
This link is much shorter and the one openrazer used.

---

The suggested changes look good to me, thanks